### DISCUSSION TITLE
Is g2 normalisation for pulsed sources sufficient?

### DISCUSSION THREAD

**Discussion by vzickus** [PicoQuant/snAPI#31] [General]
Just wondering if this normalization should include a repetition rate factor? For example,  the "coincidence" function needs a factor of repetition rate to normalise to 1 properly. 

Taking a very long correlation window using a pulsed laser seems to make the g2 near zero > 1 for a coherent source.

![image](https://github.com/PicoQuant/snAPI/assets/6878017/08b2420e-ed03-4a16-8dca-1dda8633a9ab)

**Comment by tpoint75** [comment]
Hello @vzickus  do you have any news to this?

**Reply by vzickus** [reply]
Apologies, I did not notice there was a reply @tpoint75  It seems to me that one should account for the repetition rate to normalise this properly.
